web/management/commands/generate_css_files.py

```diff
@@ -0,0 +1,43 @@
+import os
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+from pygments.formatters.html import HtmlFormatter
+
+class Command(BaseCommand):
+    help = 'Generate static CSS files from pygments styles'
+
+    def handle(self, *args, **options):
+        for style in settings.PYGMENTS_STYLES:
+            self.generate_file(style)
+
+    def generate_file(self, style):
+        css = HtmlFormatter.get_style_defs(HtmlFormatter(style=style))
+        
+        # we need to prepend the `syntax` class before every class selector in the stylesheet to make sure that the styles get applied
+        # only to our code snippet and not any other DOM element that happens to have the same class name.
+        modified_css = []
+        for line in css.split("\n"):
+            if line[0] == '.':
+                line = '.syntax ' + line
+            modified_css.append(line)
```

Comment:
Take a look at `get_style_defs` `arg` argument on https://pygments.org/docs/formatters/#HtmlFormatter
i think it can be used to do this without having to iterate over the lines ourselves.